hudi-common/src/main/java/org/apache/hudi/common/fs/FSUtils.java

```diff
@@ -474,8 +474,11 @@ public static boolean isLogFile(Path logPath) {
   }
 
   public static boolean isLogFile(String fileName) {
-    Matcher matcher = LOG_FILE_PATTERN.matcher(fileName);
-    return fileName.contains(".log") && matcher.find();
+    if (fileName.contains(".log")) {
```

Comment:
can you add a const var for ".log" and use it here.

Reply:
Addressed